.github/workflows/sonarcloud-pull.yml

```diff
@@ -4,13 +4,14 @@ on:
     workflows: ["Code Analysis"]
     types:
       - completed
-
 jobs:
+
```

Comment:
Just one thing though, I think we should restore the original spacing.

Reply:
Yes,  it is better!